components/mime.rst

```diff
@@ -0,0 +1,541 @@
+.. index::
+   single: MIME
+   single: MIME Messages
+   single: Components; MIME
+
+The Mime Component
+==================
+
+    The MIME component allows manipulating the MIME messages used to send emails.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/mime
+
+Alternatively, you can clone the `<https://github.com/symfony/mime>`_ repository.
```

Comment:
Is it still something we want to promote?

Reply:
If I am right we have this paragraph for all components, but I share your thought and maybe we don’t need to promote it that much.

Reply:
I know, my suggestion was to indeed remove this sentence everywhere.

Reply:
Yes of course, everywhere or nowhere 👍🏻

Reply:
I've created #11295 to discuss about this. Thanks.